zerver/management/commands/add_users_to_mailing_list.py

```diff
@@ -49,7 +49,7 @@ def handle(self, *args: Any, **options: str) -> None:
                                    'ZULIP_FRIENDS_LIST_ID to your server settings file.')
 
         endpoint = "https://%s.api.mailchimp.com/3.0/lists/%s/members" % \
-                   (options['api_key'].split('-')[1], options['list_id'])
+                   (options['api_key'].split('-')[1], options['list_id'])  # type: ignore # mypy is assuming `options['api_key'] is a None type.
```

Comment:
Wouldn’t this be better solved by adding a variable
```python
api_key = options['api_key']
if api_key is None:
    …
api_key.split('-')
```
for which mypy knows how remember that the `None` case has already returned?

Reply:
This certainly crossed my mind!  I'll make the changes.